controller/src/beerocks/master/db/db.cpp

```diff
@@ -1232,6 +1232,48 @@ const std::vector<beerocks_message::sWifiChannel> db::get_hostap_supported_chann
     return n->hostap->supported_channels;
 }
 
+/**
+ * @brief Add supported operating class to the database.
+ * Currently this function is a wrapper which converts the operating
```

Comment:
There should be an empty line after the @brief, otherwise the whole paragraph is considered the "brief". (nitpick)